docs/linkify.rst

```diff
@@ -324,6 +324,48 @@ instance.
    'a b c <a href="http://example.com" rel="nofollow">http://example.com</a> d e f'
 
 
+It includes optional keyword arguments to specify allowed of top-level
```

Comment:
I think this should be "to specify allowed top-level".

Reply:
Yep! Fixed.